ts/sagas/instabug.ts

```diff
@@ -26,6 +26,13 @@ function* watchInstabugSaga(): IterableIterator<Effect> {
   }
 }
 
+export function* updateInstabugBadgeSaga(): IterableIterator<Effect> {
+  const repliesCount: SagaCallReturnType<
+    typeof loadInstabugUnreadMessages
+  > = yield call(loadInstabugUnreadMessages);
+  yield put(instabugUnreadMessagesLoaded(repliesCount));
+}
+
 export default function* root(): IterableIterator<Effect> {
   yield fork(watchInstabugSaga);
```

Comment:
```suggestion
  // Update number of instabug unread messages
  yield takeLatest(updateInstabugUnreadMessages, updateInstabugBadgeSaga);
  yield fork(watchInstabugSaga);
```
_takeLatest_ is even better because if a new request is coming it discards the current one (if it is running) https://redux-saga.js.org/docs/api/#takelatestpattern-saga-args

Reply:
Perfect :)

Reply:
ok